<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyWidget</class>
 <widget class="QWidget" name="MyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MyWidget</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>691</width>
     <height>701</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="paintContainer"/>
  </widget>
  <widget class="QLabel" name="tensionL">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>平滑系数：</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="tensionSpinB">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::UpDownArrows</enum>
   </property>
   <property name="showGroupSeparator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.050000000000000</double>
   </property>
   <property name="value">
    <double>0.500000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="grainSpinB">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
   <property name="value">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="grainL">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>插值点数：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="goBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>插 值</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>原始线段</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="clearBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>清 空</string>
   </property>
  </widget>
  <widget class="QPushButton" name="moveCarBtn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>570</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>小车运动</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>370</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>显示插值点</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>520</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bodoni MT Black</family>
    </font>
   </property>
   <property name="minimum">
    <number>50</number>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
   <property name="singleStep">
    <number>2</number>
   </property>
   <property name="pageStep">
    <number>2</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="sliderPosition">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::NoTicks</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>480</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>510</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>小车速度：</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
